frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/multiple-pool.ts

```diff
@@ -0,0 +1,76 @@
+import { NS } from '../utils/consts';
+
+// Storage class form
+export const createStorageClassButton: string = 'item-create';
+
+// Create storage class from
+export const provisionerDropdown: string = 'storage-class-provisioner-dropdown';
+export const expandPvcCheckbox: string = 'expand-pvc-checkbox';
+export const storagePoolDropdown: string = 'pool-dropdown-toggle';
+export const createNewPoolButton: string = 'create-new-pool-button';
+
+// Create new pool form
+export const titleLabel: string = 'modal-title';
+export const poolNameTextBox: string = 'new-pool-name-textbox';
+export const replicaDropdown: string = 'replica-dropdown';
+export const confirmAction: string = 'confirm-action';
+export const cancelAction: string = 'modal-cancel-action';
+
+// Pool status
+export const emptyStateBody: string = 'empty-state-body';
+export const poolStatus: string = 'Ready';
+
+export const poolMessage: Map<string, string> = new Map([
+  [
+    'PROGRESS',
+    'The creation of an OCS storage cluster is still in progress or have failed please try again after the storage cluster is ready to use.',
+  ],
+  ['POOL_START', 'Pool $PoolName creation in progress'],
+  ['POOL_DUPLICATED', 'Pool "$PoolName" already exists'],
+  ['POOL_CREATED', 'Pool $PoolName was successfully created'],
+]);
```

Comment:
I find the usage of Map a bit of an overkill here. We could just use the plain JavaScript object. As I can see the keys are plain strings. I also don't see the need of maintaining insertion order.